orb/src/jobs/setup.yml

```diff
@@ -3,14 +3,14 @@ parameters:
     default: default
     type: executor
   npm-version:
-    default: '8'
+    description: This value is deprecated and no longer used. We now use Node's bundled version of npm.
+    default: ''
     type: string
 
 executor: << parameters.executor >>
 
 steps:
   - attach-workspace
-  - run: sudo npm install -g npm@<<parameters.npm-version>>
```

Comment:
**suggestion**: this is still a breaking change, let's support the `npm-version` if a user sets it for now
```yaml
  - run:
      command: |-
        if [ -n "$PARAM_NPM_VERSION" ]; then
          sudo npm install -g npm@$PARAM_NPM_VERSION
        fi
      environment:
        PARAM_NPM_VERSION: <<parameters.npm-version>>
```
(inspired by how the CircleCI's `node` [orb](https://circleci.com/developer/orbs/orb/circleci/node#orb-source) handles empty string parameters)

Reply:
Just found out about the `when` and `unless` properties for CircleCI 🤯 https://circleci.com/docs/configuration-reference/#the-when-step

It means the above code can be done with just:

```yml
  - when:
      condition: << parameters.npm-version >>
      steps:
          - run: sudo npm install -g npm@$PARAM_NPM_VERSION    
```